hack/verify-govet.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+run_individual_tests() {
+  files_to_test=$1
+  tests="printf methods structtags composites asmdecl assign atomic bool buildtags copylocks nilfunc rangeloops unreachable shadow unsafeptr unusedresult"
+
+  for test in $tests
+  do
+    touch "_output/govet/${test}.log"
+    for file in $files_to_test
+    do
+      go tool vet "-${test}=true" $file 2>&1 | sed '/exit status/d' >> "_output/govet/${test}.log" || true
+    done
+    if [ -s "_output/govet/${test}.log" ]
+    then
+      echo "FAILURE: go vet -${test} had errors: "
```

Comment:
This seems like a ton of complexity to run per-file per-test (also, it's super slow). Once we get the codebase cleaned up, we won't care about surfacing thousands of errors at once.

What if we replaced the entire thing with this:

```
dirs=$(find_files | cut --delimiter=/ --fields=1-2 | sort -u)
for dir in $dirs
do
  go tool vet -all=true $dir 2>&1
done
```